### PR TITLE
Feature/examples timepicker

### PR DESCRIPTION
**Pull request by TomVanDenBroeck**
## PR Checklist

This PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [*] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/forms/src/lib/timepicker/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() hoursPlaceholder: string;` | - | The placeholder used in the hour picker (optional, default HH). |
+| `@Input() minutesPlaceholder: string;` | - | The placeholder used in the minutes picker, (optional, default MM). |
+| `@Input() hasError: boolean;` | - | Add has-error class to input fields inside timepicker, (optional, default false). |
+| `@Input() size: TimepickerInputSize;` | - | enum: sizes of the timepicker input fields instide the timepicker, (optional, default auto). |
```
Comment (lievenvg) [inline packages/forms/src/lib/timepicker/README.md]: You mean 'Enum sizes of ' ... 'inside'?
And shouldn't the default values be mentioned in the column `Default value`?

File: packages/forms/examples/forms/pages/demo/demo.page.ts [hunk 2]
```diff
@@ -1,8 +1,39 @@
 import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
```
Comment (lievenvg) [inline packages/forms/examples/forms/pages/demo/demo.page.ts]: Not being used I presume?

File: packages/forms/src/lib/timepicker/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() hoursPlaceholder: string;` | - | The placeholder used in the hour picker (optional, default HH). |
+| `@Input() minutesPlaceholder: string;` | - | The placeholder used in the minutes picker, (optional, default MM). |
+| `@Input() hasError: boolean;` | - | Add has-error class to input fields inside timepicker, (optional, default false). |
```
Comment (lievenvg) [inline packages/forms/src/lib/timepicker/README.md]: Better to set it as `has-error`.

File: packages/forms/examples/forms/pages/demo/demo.page.ts [hunk 4]
```diff
@@ -1,8 +1,39 @@
...
 @Component({
-	templateUrl: './demo.page.html',
+    templateUrl: './demo.page.html',
 })
-export class DemoPageComponent {
+export class FormsDemoPageComponent {
+
+	public time1 = "";
+	public time2 = "";
+	public time3 = "";
```
Comment (jsebrech) [inline packages/forms/examples/forms/pages/demo/demo.page.ts]: misschien 1 van de time controls een initiele waarde meegeven?

File: packages/forms/examples/forms/forms.module.ts [hunk 5]
```diff
@@ -1,13 +1,19 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { CodeSnippetModule } from '@acpaas-ui/ngx-components/code-snippet';
+import { FormsModule } from '@angular/forms';
+import { TimepickerModule } from '@acpaas-ui/ngx-components/forms';
+import { RouterModule } from '@angular/router';
```
Comment (TriangleJuice) [inline packages/forms/examples/forms/forms.module.ts]: Why do you need the RouterModule?

File: packages/forms/examples/forms/pages/demo/demo.page.html [hunk 6]
```diff
@@ -1,3 +1,56 @@
...
+		hoursPlaceholder="20"
+		minutesPlaceholder="30"
+		size="small"
+		[(ngModel)]="time2">
+	</aui-timepicker>
+</div>
+
+<pre class="a-pre">{{ time2 }}</pre>
+
+<h3 class="h5 u-margin-bottom">error</h3>
```
Comment (TriangleJuice) [inline packages/forms/examples/forms/pages/demo/demo.page.html]: I would name this "Validation"

**Pull request merged**